### PR TITLE
zero unused constant.

### PR DESCRIPTION
**Pull request by vondele**
No functional change.

### CODE REVIEW

**Comment by snicolet** [comment]
@vondele 

I agree that we should put zero in the array rather than the random, unused value 483 (and I already suggested doing so in this comment in september : https://github.com/official-stockfish/Stockfish/commit/eccccba0ce4e2d627cbe2adb1bf4a692d595ca99 )

However it seems to me that the comment in line 71 is overkill, it would be better to suppress it. People interested in the razor_margin array will easily figure out by themselves why razor_margin[0] is zero.

**Comment by vondele** [comment]
Not sure if the comment is overkill. If one knows the code well enough, the zero might be a clear enough sign, but for people new to the code, likely not.

Certainly in the past razor_margin[0] has been tuned several times, as a quick (non-exhaustive) search suggests:

http://tests.stockfishchess.org/tests/view/566324f00ebc5902cdc08913
http://tests.stockfishchess.org/tests/view/57a487be0ebc591c761f5e4c

Anyway, no strong opinions on the comment either.

File: src/search.cpp [hunk 1]
```diff
@@ -68,7 +68,8 @@ namespace {
   const int skipPhase[] = { 0, 1, 0, 1, 2, 3, 0, 1, 2, 3, 4, 5, 0, 1, 2, 3, 4, 5, 6, 7 };
 
   // Razoring and futility margin based on depth
-  const int razor_margin[4] = { 483, 570, 603, 554 };
+  // razor_margin[0] is unused as long as depth >= ONE_PLY in search
+  const int razor_margin[4] = { 0, 570, 603, 554 };
```
Comment (mcostalba) [inline src/search.cpp]: @vondele  array size is redundant and against usual conventions, I suggest simply:

    // Razoring and futility margin based on depth (valid for depth >= ONE_PLY)
    const int razor_margin[] = { 0, 570, 603, 554 };

Comment (Rocky640) [inline src/search.cpp]: @mcostalba :  if you don't like redundant array sizes, take a look at pawn.cpp and evaluate.cpp

**Pull request closed without merging**